File: BH Tic Tac Toe IntelliJ Project/src/org/bh/game/ttt/game/AI.java
package org.bh.game.ttt.game;

import java.awt.Point;

/**
 * AI, made for BH Tic Tac Toe NetBeans Project, is copyright Blue Husky Programming ©2014 GPLv3 <hr/>
 * 
 * @author Kyli of Blue Husky Programming
 * @version 1.0.0
 *		- 2014-09-23 (1.0.0) - Kyli created AI
 * @since 2014-09-23
 */
public class AI extends Player
{
	public AI(String name, char symbol)
	{
		super(name, symbol);
	}

	@Override
	public Move nextMove()
	{
		return new Move(0, 0);
	}
}

File: BH Tic Tac Toe IntelliJ Project/src/org/bh/game/ttt/game/AI.kt
package org.bh.game.ttt.game;

/**
 * AI, made for BH Tic Tac Toe NetBeans Project, is copyright Blue Husky Software ©2014 GPLv3 <hr/>
 * 
 * @author Kyli & Ben of Blue Husky Software
 * @version 2.0.0
 *          - 2016-10-10 (2.0.0) - Ben migrated to Kotlin
 *          - 2014-09-23 (1.0.0) - Kyli created AI
 * @since 2014-09-23
 */
class AI(name: String, symbol: Char) : Player(name, symbol) {

    override fun nextMove(callback: Function1<Move, Unit>) {
        callback.invoke(Move(PositionZero, PositionZero))
    }
}
